### PR TITLE
ace 7.0.0

### PR DESCRIPTION
**Pull request by jwillemsen**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by jwillemsen** [comment]
Looks C++11 is not default enabled for clang on macosx, closing for now. Will try again with the next ACE release. Found that we force c++11 manually for github actions, something we shouldn't do, see https://github.com/DOCGroup/ACE_TAO/pull/1424

**Comment by carlocab** [comment]
>Looks C++11 is not default enabled for clang on macosx

You can enable this using `ENV.cxx11`.

**Comment by jwillemsen** [comment]
> > Looks C++11 is not default enabled for clang on macosx
> 
> You can enable this using `ENV.cxx11` at the top of the install block.

Thanks!

**Comment by carlocab** [comment]
Btw, it seems you have commits in this PR that have already been included in the master branch. Please rebase this and squash the new commits together if you can.

File: Formula/ace.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Ace < Formula
   desc "ADAPTIVE Communication Environment: OO network programming in C++"
-  homepage "https://www.dre.vanderbilt.edu/~schmidt/ACE.html"
-  url "https://github.com/DOCGroup/ACE_TAO/releases/download/ACE%2BTAO-6_5_12/ACE-6.5.12.tar.bz2"
-  sha256 "ccd94fa45df1e8bb1c901d02c0a64c1626497e5eeda2f057fcf0a1578dae2148"
+  homepage "https://github.com/DOCGroup/ACE_TAO"
```
Comment (bayandin) [inline Formula/ace.rb:3]: Do we really want to update homepage? 
GitHub repo mentions https://www.dre.vanderbilt.edu/~schmidt/TAO.html as a homepage

Comment (jwillemsen) [reply Formula/ace.rb]: Reverted that

**Comment by carlocab** [comment]
It appears to compile on ARM, but the test is failing:
```
==> ./test_callback
dyld: Library not loaded: @rpath/libACE.dylib
  Referenced from: /private/tmp/ace-test-20210125-24819-ymwylk/./test_callback
  Reason: image not found
```
I suspect this is due to the non-default prefix of `/opt/homebrew`.

**Comment by jwillemsen** [comment]
No idea why that fails, currently lack free time to dive into this in detail. This doesn't seem related to ACE 7, is there a way to see if this worked with 6.5.12?

**Comment by carlocab** [comment]
>is there a way to see if this worked with 6.5.12?

It probably doesn't. Still, we can check: https://github.com/Homebrew/homebrew-core/actions/runs/509316013

**Comment by carlocab** [comment]
[Same error](https://github.com/Homebrew/homebrew-core/runs/1761892341?check_suite_focus=true#step:6:51):
```
==> ./test_callback
dyld: Library not loaded: @rpath/libACE.dylib
  Referenced from: /private/tmp/ace-test-20210125-52340-j83pbz/./test_callback
  Reason: image not found
```

**Comment by jwillemsen** [comment]
Maybe related https://github.com/conda-forge/ace-feedstock/blob/master/recipe/fix_macos_rpath.patch

**Comment by carlocab** [comment]
It certainly looks like it. Any chance you could apply that patch here so that we can get ace bottled for ARM?

**Comment by carlocab** [comment]
The patch didn't cut it, I'm afraid. Let's just remove the patch. Bottling for ARM can wait till the next release of ACE.

**Comment by jwillemsen** [comment]
> The patch didn't cut it, I'm afraid. Let's just remove the patch. Bottling for ARM can wait till the next release of ACE.

Ok, got rid of that change again

**Review by carlocab — DISMISSED** [review]
Thanks, @jwillemsen. Please avoid merge commits on your next PR; they complicate our merge workflow.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**